doc/api/child_process.md

```diff
@@ -1464,6 +1470,34 @@ The shell should understand the `-c` switch. If the shell is `'cmd.exe'`, it
 should understand the `/d /s /c` switches and command line parsing should be
 compatible.
 
+If `shell` and `args` are both specified, the shell should understand
+[POSIX single-quotes][]. If the shell is `'cmd.exe'`, it should understand the
+double-quoting used by Windows `cmd`. If the shell is `'powershell.exe'` or
+`'pwsh'`, it should understand a powershell-compatible single quoting.
+
+Since the `command` part is literally passed to the shell, you will be
+responsible for any escaping you perform. The specific way to call a file
+as a command-line program depends on the shell:
+
+* With POSIX shell, you quote the path using the usual single-quotes. You
+  may want to prefix it with `command --` or `env --` to suppress syntax,
```

Comment:
It's a path already so these lookups should not happen anyway. Will change to "if you are instead using a command, ..." or something. Not gonna force push now, just a new commit for ya to squash.